org.eclipse.jdt.core/dom/org/eclipse/jdt/core/dom/VariableBinding.java

```diff
@@ -240,17 +240,16 @@ private JavaElement getUnresolvedJavaElement() {
 		int sourceStart;
 		int sourceLength;
 		int modifiers = 0;
-		if (localVar instanceof SingleVariableDeclaration) {
-			sourceStart = localVar.getStartPosition();
-			sourceLength = localVar.getLength();
-			final SingleVariableDeclaration singleVariableDeclaration = (SingleVariableDeclaration) localVar;
+		sourceStart = localVar.getStartPosition();
```

Comment:
Please combine the initialization of sourceStart/sourceLength with the declarations a line above for the sake of better readability.

Reply:
OK, done.